apps/tlon-mobile/index.js

```diff
@@ -1,10 +1,13 @@
 import { registerRootComponent } from 'expo';
 import 'expo-dev-client';
+import polyfill from 'react-native-polyfill-globals';
 import { TailwindProvider } from 'tailwind-rn';
 
 import App from './src/App';
 import utilities from './tailwind.json';
 
+polyfill();
```

Comment:
Maybe add a comment here about how this is necessary in order to use react native's fetch with the Urbit client's server sent events implementation.